templates/jlink/jlinked-app.yaml

```diff
@@ -41,7 +41,7 @@ objects:
 - apiVersion: image.openshift.io/v1
   kind: ImageStream
   metadata:
-    name: ${APPNAME}-ubi9-openjdk-${JDK_VERSION}-jlink
+    name: ${APPNAME}-openjdk-${JDK_VERSION}-jlink-tech-preview
```

Comment:
this didn't need to change, but, so long as it's consistent in the template, that doesn't matter. (looks consistent to me)